urbit/umbrel-app.yml

```diff
@@ -35,18 +35,4 @@ torOnly: false
 submitter: ~mopfel-winrux
 submission: https://github.com/getumbrel/umbrel/pull/1246
 releaseNotes: >-
-  - v1.15 is a revert to v1.13
-
-  From v1.13:
-
-  - Supports setting the loom size on startup. sizes are specified in exponents of 2, from 1MB (20) to 4GB (32); can differ between processes (--loom and --urth-loom); and can be set for relevant subcommands (meld, pack, next, &c).
-
-  - Improves bit-slice performance (for noun de/serialization and many atom jets) by ensuring that implementation inner loops can be vectorized.
-
-  - Adds the vile command, for exporting the keyfile from a ship
-
-  - Adds the %xray hint, for printing the bytecode of a given expression (ty ~topfet-parmed!)
-
-  - Adds the eval command, for running hoon (from stdin) without booting a ship (ty ~mopfel-winrux!)
-
-  - Adds a better error message when a ship is already running as root (ty ~dinleb-rambep!)
+Updated Urbit binary to v1.16
```

Comment:
Would you be able to populate the release notes/changes for Urbit v1.16. This would give users some context (in the Umbrel UI) as to what has changed with this app update.

Reply:
Sure thing

Reply:
Could you fix the formatting? In order to be valid yaml, this block needs to be indented. And also due to the usage of `>-` with this blocks, we need empty lines between lines of text to actually get new lines. I would do this myself, but I can't checkout this PR due to not having permissions.

Reply:
No problem 👍